### DISCUSSION TITLE
NOTE DI DEBITO/CREDITO - Sono supportate?

### DISCUSSION THREAD

**Discussion by alexgolasibill** [fattureincloud/api#328] [Q&A] [answered]
Buongiorno a tutti!
Oggi stavo lavorando a una nuova funzionalità dove avevo la necessità di estrarre note di credito/debito. Per quanto riguarda le note di credito emesse è stato abbastanza semplice perchè ho visto che bastava passare type=credit_note alla chiamata api /issued-documents.

1. Per le **Note di Credito** ricevute (quindi non emesse) invece come funziona?
3. Per le **Note di Debito** non ho trovato nulla. C'è un modo di fare un fetch delle note di debito? 

Ringrazio tutti della disponibilita!

**Comment by valmoz** [comment]
Buongiorno @alexgolasibill,
ha letto questo articolo nelle nostre guide? https://help.fattureincloud.it/help/articolo/221-documento-elettronico-nota-debito

Grazie
Mauro

**Reply by alexgolasibill** [reply]
Grazie mille della risposta @valmoz ! 
Avevo visto questa guida. Che ti spiega effettivamente come creare una nota di debito. 

Ma non ho capito come posso fare il fetch  con filtro nota di debito quando vado a chiamare le API:
 - GET /c/{company_id}/issued_documents
 - GET /c/{company_id}/received_documents
 
 Forse mi sto perdendo qualcosa! Magari quell'attributo viene ritornato e posso usare una logica sul tipo documento TDXX?
 
 Grazie mille

**Reply by valmoz** [reply, marked as answer]
Purtroppo quel campo non risulta essere tra i filtrabili, quindi non è possibile effettuare un filtro direttamenete tramite API.
L'unica soluzione sembra essere quella di tener traccia degli ID delle note di debito, oppure di effettuare una list e selezionare lato codice gli elementi della lista desiderati.

Qui di seguito alcuni link:
- [Attributi avanzati](https://developers.fattureincloud.it/docs/guides/e-invoice-xml-customisation)
- [Filter results](https://developers.fattureincloud.it/docs/basics/additional-capabilities/filter-results)

**Reply by alexgolasibill** [reply]
Ok grazie mille! Ti ringrazio per il tuo tempo e supporto!
